src/Sulu/Bundle/AdminBundle/DependencyInjection/SuluAdminExtension.php

```diff
@@ -97,6 +102,14 @@ public function load(array $configs, ContainerBuilder $container)
         $loader = new Loader\XmlFileLoader($container, new FileLocator(__DIR__ . '/../Resources/config'));
         $loader->load('services.xml');
 
+        $container->registerForAutoconfiguration(RouteProviderInterface::class)
+                  ->addTag(AddAdminPass::ADMIN_TAG)
+                  ->addTag(RemoveForeignContextServicesPass::SULU_CONTEXT_TAG, ['context' => SuluKernel::CONTEXT_ADMIN]);
+
+        $container->registerForAutoconfiguration(NavigationProviderInterface::class)
+                  ->addTag(AddAdminPass::ADMIN_TAG)
+                  ->addTag(RemoveForeignContextServicesPass::SULU_CONTEXT_TAG, ['context' => SuluKernel::CONTEXT_ADMIN]);
```

Comment:
Wouldn't you use the `Admin` class for that? The `RouteProviderInterface` doesn't work standalone, because some types in the system are messed up....

Reply:
@danrot fixed